hw/top_earlgrey/BUILD

```diff
@@ -470,9 +470,12 @@ silicon(
     base = ":silicon_owner_sival_rom_ext",
     ecdsa_key = CLEAR_KEY_SET,
     exec_env = "silicon_owner_prodc_rom_ext",
+    # The default ROM_EXT needs to be updated to use the following target once
+    # available:
+    # //sw/device/silicon_creator/rom_ext/prodc/binaries:rom_ext_real_prod_signed_slot_a.
     rom_ext = select({
-        "//signing:test_keys": "//sw/device/silicon_creator/rom_ext/prodc:rom_ext_fake_prod_signed_slot_a",
-        "//conditions:default": "//sw/device/silicon_creator/rom_ext/prodc/binaries:rom_ext_real_prod_signed_slot_a",
+        "//signing:test_keys": "//sw/device/silicon_creator/rom_ext:rom_ext_slot_a",
```

Comment:
The `test_keys` option should probably still refer to `//sw/device/silicon_creator/rom_ext/prodc:rom_ext_fake_prod_signed_slot_a`.

That build of the ROM_EXT has the `prodc` manifest binding and has the application keys for the `prodc` customer linked into the ROM_EXT.  That allows the customer to run the fake-signed ROM_EXT on an FPGA but still boot their application image.